addons/percentage-stats/userscript.js

```diff
@@ -0,0 +1,77 @@
+export default async function ({ addon, console }) {
+  function getStats() {
+    const stats = {
+      views: els.views.innerHTML,
+      loves: els.loves.innerHTML,
+      favs: els.favs.innerHTML,
+      remixes: els.remixes.innerHTML,
+    };
+    return {
+      loves: Math.round((stats.loves / stats.views) * 100) + "%",
+      favs: Math.round((stats.favs / stats.views) * 100) + "%",
+      remixes: Math.round((stats.remixes / stats.views) * 100) + "%",
```

Comment:
Add a decimal place to prevent any confusion about why the percentage is an integer
```suggestion
      loves: (Math.round((stats.loves / stats.views) * 1000) / 10) + "%",
      favs: (Math.round((stats.favs / stats.views) * 1000) / 10) + "%",
      remixes: (Math.round((stats.remixes / stats.views) * 1000) / 10) + "%",
```